tuna/corrupt_configs.py

```diff
@@ -28,17 +28,17 @@
 import sys
 import logging
 
+from typing import Any, Union, Dict, Optional
+from io import TextIOWrapper
 from tuna.sql import DbCursor
 from tuna.metadata import TABLE_COLS_FUSION_MAP
 from tuna.metadata import TABLE_COLS_CONV_MAP
 
-LOGGER = get_logger()
-
 
-def get_logger():
+def get_logger() -> logging.Logger:
   """Setting up logger"""
-  logger = logging.getLogger('CorruptConfigs')
-  hdlr = logging.FileHandler('corrupt_configs.log')
+  logger: logging.Logger = logging.getLogger('CorruptConfigs')
```

Comment:
It's lines like this that make me question this explicit type syntax. It's more verbose, but not any more enlightening. Does make for a good tongue twister though. @JehandadKhan